internal/kibana/dashboards.go

```diff
@@ -20,8 +23,68 @@ type exportedType struct {
 	Objects []common.MapStr `json:"objects"`
 }
 
-// Export method exports selected dashboards using the Kibana Export API.
+// Export method exports selected dashboards using the Kibana APIs.
 func (c *Client) Export(dashboardIDs []string) ([]common.MapStr, error) {
+	if c.semver.LessThan(semver.MustParse("8.11.0")) {
+		return c.exportWithDashboardsAPI(dashboardIDs)
+	}
+
+	return c.exportWithSavedObjectsAPI(dashboardIDs)
+}
+
+type exportSavedObjectsRequest struct {
+	ExcludeExportDetails  bool                              `json:"excludeExportDetails"`
+	IncludeReferencesDeep bool                              `json:"includeReferencesDeep"`
+	Objects               []exportSavedObjectsRequestObject `json:"objects"`
+}
+
+type exportSavedObjectsRequestObject struct {
+	ID   string `json:"id"`
+	Type string `json:"type"`
+}
+
+func (c *Client) exportWithSavedObjectsAPI(dashboardIDs []string) ([]common.MapStr, error) {
+	logger.Debug("Export dashboards using the Kibana Saved Objects Export API")
+
+	exportRequest := exportSavedObjectsRequest{
+		ExcludeExportDetails:  true,
+		IncludeReferencesDeep: true,
+	}
+	for _, dashboardID := range dashboardIDs {
+		exportRequest.Objects = append(exportRequest.Objects, exportSavedObjectsRequestObject{
+			ID:   dashboardID,
+			Type: "dashboard",
+		})
+	}
+
+	body, err := json.Marshal(exportRequest)
+	if err != nil {
+		return nil, err
+	}
+
+	path := SavedObjectsAPI + "/_export"
+	statusCode, respBody, err := c.SendRequest(http.MethodPost, path, body)
+	if err != nil {
+		return nil, fmt.Errorf("could not export dashboards; API status code = %d; response body = %s: %w", statusCode, respBody, err)
+	}
+
+	var dashboards []common.MapStr
+	decoder := json.NewDecoder(bytes.NewReader(respBody))
+
+	for decoder.More() {
+		var dashboard common.MapStr
+		err := decoder.Decode(&dashboard)
+		if err != nil {
+			return nil, fmt.Errorf("unmarshalling response failed (body: \n%s): %w", respBody, err)
+		}
+
+		dashboards = append(dashboards, dashboard)
+	}
+
+	return dashboards, nil
```

Comment:
To reuse the code introduced in https://github.com/elastic/elastic-package/pull/1565 if finally merged.